libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -934,16 +934,21 @@ void ofFbo::updateTexture(int attachmentPoint) {
 			glPushAttrib(GL_COLOR_BUFFER_BIT);
 		}
 
-		bind();
+		// Because we temporarily change global Fbo binding state
+		// when blitting from one framebuffer to the other, we first have 
+		// to query the renderer for its binding state, and store it.
+		GLuint rendererFboBindingState = ofGetGLRenderer()->getCurrentFramebufferId();
```

Comment:
-> @procedural and this is exactly why it is sensible to review even small contributions by experts.